src/components/Header/Header.jsx

```diff
@@ -8,7 +8,11 @@ function Header() {
             marginTop:"250px",
             backgroundColor:"#1f1e1e55"
         }}>
-             <h1 className='title_header'>Your Review/Suggestion for this Project</h1>
+             
+             
+            <h1 class="mb-4 font-extrabold text-gray-900 dark:text-white "><span class="text-transparent bg-clip-text bg-gradient-to-r to-emerald-600 from-sky-400">For XF </span>Your Feedback </h1>
```

Comment:
in. react there is no class replace with className and this code look like it is tailwind css